src/helm/benchmark/scenarios/audio_language/audiocaps_senario.py

```diff
@@ -35,28 +35,23 @@ class AudioCapsScenario(Scenario):
         }
     """
 
+    DOWNLOADING_URL = "https://huggingface.co/datasets/Olivia714/audiocaps/resolve/main/wav_files.zip"
+    REFERENCE_URL = "https://huggingface.co/datasets/Olivia714/audiocaps/resolve/main/test.csv"
+
     name = "audiocaps"
     description = "A large-scale dataset of about 46K audio clips to human-written text pairs \
         ([Kim et al, 2019](https://aclanthology.org/N19-1011.pdf))."
-    tags = ["audio", "captioning"]
+    tags: List[str] = ["audio", "captioning"]
 
     def get_instances(self, output_path: str) -> List[Instance]:
         instances: List[Instance] = []
         data_dir: str = os.path.join(output_path, "wav_files")
-
-        ensure_file_downloaded(
-            source_url="https://huggingface.co/datasets/Olivia714/audiocaps/resolve/main/wav_files.zip",
-            target_path=data_dir,
-            unpack=True,
-        )
-
-        assert os.path.exists(data_dir), "Download the wav_files from Olivia714/audiocaps"
-
-        for row in tqdm(load_dataset("Olivia714/audiocaps", cache_dir=output_path, split=TEST_SPLIT)):
+        ensure_file_downloaded(source_url=AudioCapsScenario.DOWNLOADING_URL, target_path=data_dir, unpack=True)
+        assert os.path.exists(data_dir), f"Download the wav_files from {AudioCapsScenario.DOWNLOADING_URL}"
```

Comment:
Could you delete this line? This is redundant since we're using `ensure_file_downloaded`